packages/config/config/devices/0x0346/alarm_keypad_gen2.json

```diff
@@ -326,8 +308,8 @@
 			"allowManualEntry": true
 		},
 		"22": {
-			"label": "System Security Mode Display - 601 = Always On,1 - 600 = periodic interval, 0 = Always Off, except when there's activity",
-			"description": "System security mode display options",
+			"label": "System Security Mode Display",
+			"description": "Options are: 601 = Always On, 1 - 600 = periodic interval, 0 = Always Off, except when there's activity",
```

Comment:
0 and 601 should be added as predefined `"options"` instead. Also, this sounds like there should be a unit.

Reply:
Cool, I added the options but what about 1 - 600? I also can't see a unit mentioned on the PDF.

Reply:
Yup, just leave it as manual entry. 0 and 601 will appear as suggestions in UIs which support this.
Since there doesn't seem to be a unit, the description or label should somehow mention that this configures an interval